Python/ceval.c

```diff
@@ -4555,8 +4555,15 @@ maybe_call_line_trace(Py_tracefunc func, PyObject *obj,
 void
 PyEval_SetProfile(Py_tracefunc func, PyObject *arg)
 {
-    PyThreadState *tstate = _PyThreadState_GET();
-    PyObject *temp = tstate->c_profileobj;
+    PyThreadState *tstate;
+    PyObject *temp;
```

Comment:
Since Python 3.6, we use C99 and so it's fine to add code before variable declarations.